packages/api-explorer/src/Doc.jsx

```diff
@@ -239,6 +243,21 @@ class Doc extends React.Component {
     );
   }
 
+  renderLogs() {
+    if (!this.props.flags.apilogs) return null;
+
+    return (
+      <Logs
```

Comment:
Do you mean you want to pass it down from the root component? So when we are instantiating the Api-Explorer, we pass it in as a parameter?

Reply:
Yeah that's what I had in mind! I dont really mind about it being this way, but if it was external it would mean it wouldn't have to always get bundled in with the api-explorer code. What'dya think?

Reply:
Done